dlk/tests/test_optimizer.py

```diff
@@ -13,16 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # =============================================================================
-"""Test file for GraphRunner."""
+"""Test file for Optimizer."""
 import unittest
 from core.data_types import Float32, Uint32, Int32, QUANTIZED_NOT_PACKED
-from core.graph import Graph, GraphRunner
-from core.optimizer import Optimizer
+from core.optimizer import pass_remove_identities, pass_transpose, pass_precompute, \
+    pass_propagate_quantization_details_into_conv, pass_compute_thresholds, pass_pack_weights, \
+    pass_quantize_convolutions, pass_propagate_datatypes, pass_propagate_output_type_backward, pass_dot_graph
+from core.graph import Graph
 from core.operators import Add, AveragePool, BatchNormalization, Constant, Conv, Identity, Input, \
     MaxPool, Operator, Output, Transpose, QTZ_binary_mean_scaling, QTZ_linear_mid_tread_half, Reshape, Softmax
 
 import numpy as np
-from typing import Any, Dict, List, Tuple
+from typing import Tuple
 
 
 class TestOptimizer(unittest.TestCase):
```

Comment:
I didn't understand what is verified in this test. Is it not unit test ?
I think need to add unit tests for each passes, and some public graph matcher functions and classes.

Reply:
@antonionevado @nlpng How is this?